scripts/release/release.sh

```diff
@@ -0,0 +1,168 @@
+#!/bin/sh
+
+SCRIPTS_DIR=$(dirname "$0")
+BASE_DIR="$(pwd)"
+SRC_DIR="$BASE_DIR/libelektra"
+BUILD_DIR="$SRC_DIR/build"
+FTP_DIR="$BASE_DIR/ftp"
+DOC_DIR="$BASE_DIR/docu"
+
+install_elektra() {
+	echo "Installing elektra..."
+	rm -rf $BUILD_DIR && mkdir $BUILD_DIR && cd $BUILD_DIR && cmake -DBUILD_SHARED=ON -DBUILD_FULL=ON -DBUILD_STATIC=ON ..
+	cd ..
+	cmake --build build -- -j5
+	cd $BUILD_DIR
+	make install
+	export VERSION=$(kdb get system/elektra/version/constants/KDB_VERSION)
+	export DVERSION=$VERSION-1
+}
+
+run_updates() {
+	echo "Running updates..."
+	mkdir $BASE_DIR/$VERSION
+	# regenerate dot of plugins
+	$SRC_DIR/scripts/dev/draw-all-plugins > $BASE_DIR/$VERSION/draw-all-plugins                         # TODO: rename file
```

Comment:
Also do not forget to store the error log of all these things (which is more important than stdout).

Reply:
Should stderr be combined with stdout or should the stderr be output to a dedicated error log file?

Reply:
In general I prefer to have it separated as it makes reviewing of logs much easier. But in some cases it is not clear to which part an error refers, e.g. for the test case runs, and then the combined one is better. In these cases we need to triple check that silent errors are impossible (i.e. the build job will abort).